caffe2/utils/proto_utils.cc

```diff
@@ -156,6 +166,14 @@ bool ParseFromString(const string& spec, Message* proto) {
 }
 } // namespace TextFormat
 
+bool ParseProtobufFromLargeString(const string& str, Message* proto) {
+  ::google::protobuf::io::ArrayInputStream input_stream(str.data(), str.size());
```

Comment:
Nit: do you want to add and use

```
using ::google::protobuf::io::ArrayInputStream;
using ::google::protobuf::io::CodedInputStream;
```
above and get rid of the long references here?

Reply:
It's only used once so I'll probably just keep it this way?